website/docs/r/cm_network.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: network"
+sidebar_current: "docs-rightscale-resource-network"
+description: |-
+  Create and maintain a rightscale cloud management network.
+---
+
+# rightscale_network
+
+Use this resource to create, update or destroy rightscale [networks](http://docs.rightscale.com/cm/dashboard/manage/networks/index.html) in cloud management.
+
+## Example Usage
+
+```hcl
+resource "rightscale_network" "us-oregon-devops-vpc" {
+  name = "us-oregon-devops-vpc"
+  description = "AWS US Oregon vpc for devopery"
+  cloud_href = "/api/clouds/6"
+  cidr_block = "192.168.0.0/16"
+}
+
+output "us-oregon-devops-vpc-uuid" {
+  value = "${rightscale_network.us-oregon-devops-vpc.resource_uid}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cloud_href` - (Required) Cloud you want to create the network in.
+
+* `cidr_block` - (Optional*) Cloud-specific.  Some clouds require this field, others do not.
+
+* `name` - (Optional) Network name.
+
+* `description` - (Optional) Network description.
+
+* `instance_tenancy` - (Optional) Launch policy for AWS instances in the network. Specify 'dedicated' to force all instances to be launched as 'dedicated'.  Defaults to 'default.'
+
+* `route_table_href` - (Optional) Sets the default route table for this network, useful if you create the route table with a different resource.
```

Comment:
Same as `cloud_href` and `deployment_href` - would be nice to explain how to retrieve these. Maybe the easiest way is with `rsc`. So something like:
```
xxx hrefs can be retrieved using the RightScale API command line client `rsc` [TBD: link goes here]. Example: 
    rsc cm15  --xm 'object:has(.rel:val("self")).href' index /api/deployments [TBD: replace deployements appropriately]
```

Reply:
Both are top level documented resources or datasources already - still need links IN this doc section?  (https://github.com/rightscale/terraform-provider-rightscale/blob/master/website/docs/d/cm_cloud.markdown#example-usage && https://github.com/rightscale/terraform-provider-rightscale/blob/master/website/docs/d/cm_deployment.markdown#example-usage-get-existing-deployment-href)

Reply:
Makes sense to use datasources instead of `rsc` so maybe it's fine as is. I guess the description could say `href of ...` so it's clear that you can get to is using a datasource and extracting the href of the returned resource.